frontend/src/pages/MapView/index.jsx

```diff
@@ -33,28 +47,30 @@ class MapView extends React.Component {
 
   render() {
     return (
-      <ReactMapGL
-        {...this.state.viewport}
-        width="100%"
-        height="100vh"
-        onViewportChange={viewport => this.setState({ viewport })}
-        mapboxApiAccessToken={process.env.REACT_APP_MAPBOX_ACCESS_TOKEN}
-      >
-        <Search />
-        {this.props.markers.map(this.renderMarkers)}
-        {this.state.popup && (
-          <Popup
-            latitude={this.state.popup.latitude}
-            longitude={this.state.popup.longitude}
-            tipSize={5}
-            anchor="top"
-            closeOnClick={false}
-            onClose={() => this.setState({ popup: null })}
-          >
-            <p>{this.state.popup.name}</p>
-          </Popup>
-        )}
-      </ReactMapGL>
+      <div>
+        <Search searchResults={searchResults} />
+        <ReactMapGL
+          {...this.state.viewport}
+          width="100%"
```

Comment:
Can we bring this into the stylesheets? If it's too challenging, that's okay, but it's nice to avoid using inline styling wherever possible.